BraintreePSP/BraintreePaymentScreensBuilder.swift

```diff
@@ -25,12 +25,19 @@ final public class BraintreePaymentScreenBuilder: PaymentScreenBuilder {
     public init(){}
 
     public func buildAddCardScreen(
+        allowToSaveCard: Bool,
         paymentsToken: PaymentSDKToken,
         paymentMethodAdded: ScreenResultCallback<Nonce>?,
         flowItemCallback: ScreenResultCallback<Screen>?
     ) {
         let request = BTDropInRequest()
         request.uiCustomization = getUiCustomization()
+        request.cardholderNameSetting = .required
+        if allowToSaveCard {
+            request.allowVaultCardOverride = true
```

Comment:
This looks a bit weird. Shouldn't request.vaultCard be set in both cases? Like this:
```
if allowToSaveCard {
            request.vaultCard = true
            request.allowVaultCardOverride = true
} else {
            request.vaultCard = false
        }
```

Reply:
According to docs (https://braintree.github.io/braintree-ios-drop-in/current/Classes/BTDropInRequest.html) all variables have default value:
vaultCard is true by default
and 
allowVaultCardOverride is false. 
So we override this value only when required.